apps/trackers/common.py

```diff
@@ -329,6 +333,19 @@ def _description_embargoed(self):
 
         return description_parts
 
+    def _description_contract_priority(self):
```

Comment:
Can this be a static method?

Reply:
sure, I've changed all methods that can be static in this class, in a new commit, please take a look again.